### ISSUE TITLE
Twitter rel=me verification not working

### ISSUE THREAD

**Issue by aaronpk** [indieweb/indiewebify-me#59]
for http://eddiehinkle.com/

https://indiewebify.me/validate-rel-me/?url=http%3A%2F%2Feddiehinkle.com

**Comment by kevinmarks** [comment]
the amended endpoint I use for verification works:

https://indiewebify.me/rel-me-links-info/?url1=http://eddiehinkle.com/&url2=https://twitter.com/eddiehinkle

but the old one fails:

https://indiewebify.me/rel-me-links/?url1=http://eddiehinkle.com/&url2=https://twitter.com/eddiehinkle

**Comment by kevinmarks** [comment]
DIffernce between the 2 is:

````
	list($profileUrl, $secure, $previous) = IndieWeb\relMeDocumentUrl($url2);
	if (!$secure)
		return crossOriginResponse("Inbound rel-me URL redirects insecurely" . print_r($previous, true), 400);
````
My fn ignores this 'secure' flag - which is triggered by the t.co link being https and the users' site being http only.

**Comment by gRegorLove** [comment]
Think this is part of #54

**Comment by aaronpk** [comment]
Fixed in #64
